operator/pkg/tasks/init/karmadaresource.go

```diff
@@ -68,7 +68,7 @@ func runSystemNamespace(r workflow.RunData) error {
 
 	err := apiclient.CreateNamespace(data.KarmadaClient(), &corev1.Namespace{
 		ObjectMeta: metav1.ObjectMeta{
-			Name: data.GetNamespace(),
+			Name: constants.KarmadaSystemNamespace,
```

Comment:
The following two places still referencing the names with `data.GetNamespace`, I guess should be updated accordingly, right?
https://github.com/karmada-io/karmada/blob/8d1a8007ebdbd97a2fe12cf8ee7d4694cfb2ed82/operator/pkg/tasks/init/karmadaresource.go#L75

https://github.com/karmada-io/karmada/blob/8d1a8007ebdbd97a2fe12cf8ee7d4694cfb2ed82/operator/pkg/tasks/init/karmadaresource.go#L78

Reply:
Yes, that's right. the namespace message of these logs should be consistent. 
updated, thanks.